modules/deploy/pages/deployment-option/self-hosted/kubernetes/aks-guide.adoc

```diff
@@ -43,18 +44,78 @@ NOTE: The Helm chart configures `podAntiAffinity` rules to make sure that only o
 az group create --name redpandaResourceGroup --location eastus
 ----
 
-. Create an AKS cluster with three worker nodes (one for each Redpanda broker):
+. Create an AKS cluster:
 +
 [,bash]
 ----
-az aks create -g redpandaResourceGroup -n redpanda \
+az aks create -g redpandaResourceGroup -n <cluster-name> \
   --node-count 3 \
   --generate-ssh-keys \
   --enable-node-public-ip
+  --node-vm-size Standard_L8s_v3 \
+  --disable-file-driver
 ----
 +
 TIP: For all available options, see the https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-create[AKS documentation^].
 
+=== Create a StorageClass for your local NVMe disks
+
+When you provisioned the AKS cluster, you selected an instance type that comes with local NVMe disks. However, these disks are not automatically mounted or formatted upon creation. To use these local NVMe disks, you must mount and format them, and you must create the necessary PersistentVolumes (PVs). To automate this process, you can use a Container Storage Interface (CSI) driver.
+
+In this step, you install the recommended https://github.com/metal-stack/csi-driver-lvm[local volume manager (LVM) CSI driver]. Then, you create a StorageClass that references the LVM CSI driver and specifies the recommended XFS filesystem.
+
+. Install the LVM CSI driver:
++
+[,yaml,lines=6]
+----
+helm repo add metal-stack https://helm.metal-stack.io
+helm repo update
+helm install csi-driver-lvm metal-stack/csi-driver-lvm \
+  --namespace csi-driver-lvm \
+  --create-namespace \
+  --set lvm.devicePattern='/dev/nvme[0-9]n[0-9]'
+----
++
+The `lvm.devicePattern` property specifies the pattern that the CSI driver uses to identify available NVMe volumes on your worker nodes.
+
+. Create the StorageClass:
++
+.`csi-driver-lvm-linear-xfs.yaml`
+[,yaml,lines=5-8+10-11]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-driver-lvm-linear-xfs
+provisioner: lvm.csi.metal-stack.io
+reclaimPolicy: Retain
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+parameters:
+  type: "linear"
```

Comment:
Don't we want striped to spread the iops between devices?

Reply:
cc @pfcurtis